### PR TITLE
handle NaN in tables sent via `add_table`

### PR DESCRIPTION
**Pull request by ManonMarchand**
This PR replaces NaNs by " " in tables sent to the widget. Might need discussion, not very satisfying solution :/

Solves #42 and #47 though

### CODE REVIEW

File: js/lib/jupyter-aladin.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const loadScript = (FILE_URL, async = true, type = "text/javascript") => {
         }
     });
 };
-var AladinLiteJS_Loader = loadScript("https://code.jquery.com/jquery-3.6.1.min.js")
+let AladinLiteJS_Loader = loadScript("https://code.jquery.com/jquery-3.6.1.min.js")
```
Comment (bmatthieu3) [inline js/lib/jupyter-aladin.js:33]: Just a comment that we do not need to load jQuery anymore as it is bundled in the aladin.js loaded afterwards. I think that should be done in a separate commit.

Comment (ManonMarchand) [reply js/lib/jupyter-aladin.js:33]: Yup, you want to push that to the dev branch ? Or do I do it ?

File: ipyaladin/aladin_widget.py [hunk 2]
```diff
@@ -208,7 +233,13 @@ def add_table(self, table):
                 if isinstance(item, bytes):
                     row_data.append(item.decode('utf-8'))
                 else:
+                    if not isinstance(item, str):
+                        # replace NaN by " ", this is a quick fix 
+                        # and should be questioned when doing a rework
+                        # of this function
+                        item = " " if math.isnan(item) else item 
                     row_data.append(item)
+
```
Comment (bmatthieu3) [inline ipyaladin/aladin_widget.py:242]: I do not like it either but if it solves issues without so much performance cost, I am for it.

I just ask myself if it would not be possible to do that from the numpy array table_array directly instead of looping in python over all the columns and then all the rows.

File: js/lib/jupyter-aladin.js (hunk 1 above)
Comment (bmatthieu3) [reply js/lib/jupyter-aladin.js:33]: I can remove that in a separate commit
